api/types.go

```diff
@@ -124,7 +124,8 @@ type Options struct {
 	MirostatTau      float32 `json:"mirostat_tau,omitempty"`
 	MirostatEta      float32 `json:"mirostat_eta,omitempty"`
 
-	NumThread int `json:"num_thread,omitempty"`
+	NumThread     int    `json:"num_thread,omitempty"`
+	StopCondition []byte `json:"stop_condition,omitempty"`
```

Comment:
Very cool - do we want to make this a directive in the `Modelfile`? (maybe that's for a future PR)

Reply:
I changed it to string so it shouldn't be a problem anymore